README.md

```diff
@@ -40,10 +40,15 @@ Feel free to follow the next action points:
 * Generate your private key and certificate request: `./easyvpn request <your-jenkins-username>`
   Your private key will be generated in `./cert/pki/private`
 
-  > ### This key **must** remain **secret**!
+  > ⚠️ This key **must** remain **secret**! ⚠️
+
+* Open an issue on [jenkins-infra/helpdesk](https://github.com/jenkins-infra/helpdesk) describing the reason why you need an access to the VPN
+* Create a new pull request on [jenkins-infra/docker-openvpn](https://github.com/jenkins-infra/docker-openvpn)
```

Comment:
OK for the issue first, here is my suggestion then:

```suggestion
* Open an issue on [jenkins-infra/helpdesk](https://github.com/jenkins-infra/helpdesk) describing the reason why you need an access to the VPN
* Create a new pull request on [jenkins-infra/docker-openvpn](https://github.com/jenkins-infra/docker-openvpn) with a link to the issue previously created
```

Reply:
I've moved the "issue" bullet earlier as discussed together